### DISCUSSION TITLE
Shared element transition between screens with Voyager

### DISCUSSION THREAD

**Discussion by shriharsha-srb** [adrielcafe/voyager#263] [General]
I have looked at the [available transitions](https://github.com/adrielcafe/voyager/tree/main/voyager-transitions/src/commonMain/kotlin/cafe/adriel/voyager/transitions) and I really want to have Shared element transition of an image between 2 screens. 

Is it possible with Voyager library? Any examples to which I can refer to create such custom transition.

**Comment by bmc08gt** [comment]
Check out Orbital

**Comment by Kashif-E** [comment]
@bmc08gt can you elaborate?

**Comment by vtcosmin** [comment]
Tried to use Orbital, it can be done but not really easy to use, [code](https://github.com/VisanTudorCosmin/VoyagerSharedElementTransition/blob/master/src/jvmMain/kotlin/Main.kt)
![sharedelement](https://github.com/adrielcafe/voyager/assets/102170373/4e381075-dffa-44b1-9dcc-99ab5afbc450)

**Comment by bmc08gt** [comment]
Using it in my airbnb sample -  https://x.com/brandonmcansh/status/1727402488876126298?s=46

search bar to search page
